packages/python/plotly/plotly/tests/test_core/test_px/test_px_functions.py

```diff
@@ -329,28 +331,6 @@ def test_parcats_dimensions_max():
 
     # default behaviour
     fig = px.parallel_categories(df)
-    assert [d.label for d in fig.data[0].dimensions] == [
```

Comment:
why this deletion?

Reply:
hum good catch, this is not wanted. Let me put it back.